postgres-appliance/Dockerfile

```diff
@@ -81,35 +102,36 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && curl -s -L https://github.com/zserge/jsmn/archive/$JSMN_COMMIT.tar.gz | tar xz \
     && rm -fr jsmn && mv jsmn-$JSMN_COMMIT jsmn \
     && make install \
-    && cd .. \
+    && cd ../.. \
 
     # Clean up
-    && apt-get purge -y ${BUILD_PACKAGES} \
+    && apt-get purge -y equivs fakeroot libfakeroot libmpc3 bsdmainutils libcc1-0 groff-base libunistring0 libmagic1 file ${BUILD_PACKAGES} \
```

Comment:
"fakeroot libfakeroot libmpc3 bsdmainutils libcc1-0 groff-base libunistring0 libmagic1 file" -- were pulled by equivs and somehow autoremove doesn't want to clean them up...